Data/Vector.hs

```diff
@@ -1063,6 +1064,18 @@ forM_ :: Monad m => Vector a -> (a -> m b) -> m ()
 {-# INLINE forM_ #-}
 forM_ = G.forM_
 
+-- | /O(n)/ Apply the monadic action to all elements and their indexes of the vector, yielding a
+-- vector of results. Equivalent to @flip 'imapM'@.
```

Comment:
This is nitpicking, but I suggest `@flip@ 'imapM'`or even better `'flip' 'imapM'`.

Reply:
Fixed